server/websocket_connector_test.go

```diff
@@ -19,7 +19,7 @@ var aTestMessage = &guble.Message{
 	Body: []byte("Test"),
 }
 
-func Test_WSHandler_SubscribeAndUnsubscribe(t *testing.T) {
+func TestWebSocket_SubscribeAndUnsubscribe(t *testing.T) {
```

Comment:
So far, the convention was Test_<Type>_<Description>
Changing to Test<Type>_<Description> is ok for me if you prefer it, but should be done consistently in all test.